### ISSUE TITLE
problems with spaces in computer names?

### ISSUE THREAD

**Issue by Pomax** [symless/synergy#5977]
### Operating Systems ###

Server: Windows 10 Pro x64
Client: OSX 10.11.6

### Synergy Version ###

Synergy Pro 1.8.8.stable-25a8cb2

### Steps to reproduce bug ###

0. Buy and install same version (downloaded 20 minutes ago) on both machines.
1. Have the application set to run as server run in Windows 10 called "Yuki"
2. Have the application set to run as client on a Macbook called "Macbook Panda"
3. try to connect from client to server
4. client reports `error: server refused client with name "Macbook Panda"`
5. server reports `WARNING: unrecognised client name "MozBook Panda", check server config`
6. server config does not let me whitelist "MozBook Panda" because ... it looks like I can't add "screens" with spaces in the name? That feels like it doesn't make a lot of sense. Help?

**Comment by yupi2** [comment]
You have a `MozBook Panda` and a `Macbook Panda`. Are you sure you have the right names?

**Comment by Pomax** [comment]
No, those are the same string, I typoed one of them.

**Comment by yupi2** [comment]
Yep, spaces are blocked in the GUI [here](https://github.com/symless/synergy/blob/c3c0913633041584fa41180640d2e4c83fa92820/src/gui/src/ScreenSettingsDialog.cpp#L33).

You can find wherever the .ini config file is located and then manually changing the screen-name to have a space might work.

**Comment by Pomax** [comment]
I see. That feels like a thing that should probably be changed, as unix and OSX machines can have spaces in their name.

**Comment by Pomax** [comment]
I also cannot seem to find an .ini file int he usual places (not in the application dir, which makes sense because it's UAC locked, but also not in the local and roaming user appData dirs). Does anyone know where to find this config file?

**Comment by XinyuHou** [comment]
@Pomax 
Unfortunately, current version doesn't support space in the middle of screen name. Even after you manually change the configuration file. 

I will mark this as an enhancement.

**Comment by Pomax** [comment]
@XinyuHou thank you. I guess I'll have to wait until the next version then, renaming my computers is not really an option right now.

**Comment by loweryaustin** [comment]
FYI You don't have to rename the computer. You can simply edit the screen name. To do this, edit the settings of the client, then update the server configuration to match the string that you chose for the client.
